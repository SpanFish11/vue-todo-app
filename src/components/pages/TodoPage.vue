<script setup>
import MainContent from '@/components/templates/MainContent.vue'
</script>

<template>
  <MainContent />
</template>

<style lang="scss">
#app {
  background-image: linear-gradient(
      142deg,
      rgba(189, 58, 208, 0.7198529069831057) 30%,
      rgba(153, 90, 194, 0.5237744756105567) 55%,
      rgba(38, 121, 180, 0.6526260162268032) 79%
    ),
    url('../../assets/images/header.png');
  background-size: cover;
  background-repeat: no-repeat;
  width: 100%;
  height: 450px;
}
</style>
