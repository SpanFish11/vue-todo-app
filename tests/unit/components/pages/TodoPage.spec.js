import { beforeEach, describe, expect, it } from '@jest/globals'
import { shallowMount } from '@vue/test-utils'
import TodoPage from '@/components/pages/TodoPage.vue'

describe('Given the TodoPage component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(TodoPage, {})
  })

  it('should render', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
