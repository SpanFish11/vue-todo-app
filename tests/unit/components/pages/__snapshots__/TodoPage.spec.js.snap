// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the TodoPage component should render 1`] = `<main-content-stub />`;
